code/package.json

```diff
@@ -10,7 +10,9 @@
     "eslint-plugin-jsx-a11y": "^6.6.1",
     "eslint-plugin-react": "^7.30.1",
     "eslint-plugin-react-hooks": "^4.6.0",
+    "js-confetti": "^0.10.2",
     "react": "^18.2.0",
```

Comment:
The confetti is a fun thing! Couldn't first understand where it came from... Good job solving this! :)